libs/core/src/lib/bar/button-bar/button-bar.component.ts

```diff
@@ -24,7 +24,7 @@ import { ContentDensityService } from '@fundamental-ngx/core/utils';
               [fdType]="fdType"
               [label]="label"
               [attr.title]="ariaLabel"
-              [attr.aria-label]="ariaLabel"
+              [ariaLabel]="ariaLabel"
```

Comment:
why was this changed? We are working on the native button so `[attr.aria-label]` should work fine. You can just pass `fdType` here itself, if it was specified.

Reply:
on introducing the change only in the button bar component will introduce a breaking change since it extends the baseButton class, in order to accomodate this breaking chagne this change was necessary